app/models/page.rb

```diff
@@ -190,6 +190,10 @@ def subscriptions_count
     braintree_subscriptions.count + go_cardless_subscriptions.count
   end
 
+  def donation_followup?
+    follow_up_liquid_layout.try(:title).to_s.include?('donat')
+  end
+
```

Comment:
@shivashankar-f22labs Can you explain what this is for?

Reply:
just read the ticket. I understand now why you want to know if this is a followup action.

Reply:
ok.